apps/admin/backend/package.json

```diff
@@ -27,17 +27,6 @@
     "test:watch": "TZ=America/Anchorage jest --watch",
     "type-check": "tsc --build"
   },
-  "lint-staged": {
-    "*.+(css|graphql|json|less|md|mdx|sass|scss|yaml|yml)": [
-      "prettier --write"
-    ],
-    "*.+(js|jsx|ts|tsx)": [
-      "eslint --quiet --fix"
-    ],
```

Comment:
How was prettier being run as part of eslint before? I see how the new pattern now runs it on JS/TS files, but I can't tell how it was being run on those files before, so I'm having trouble evaluating how we know it's no longer being run as part of eslint.

Reply:
I went looking for the answer and realized that we were still including `eslint-plugin-prettier` in `vx/react` contexts. I fixed that and removed `eslint-plugin-prettier` in [4cb71cc](https://github.com/votingworks/vxsuite/pull/5576/commits/4cb71cc1cd9ebb3942827e8b52369e8179612e06). The same change was made in `vx/recommended` to disable prettier within eslint.